Library/Homebrew/cmd/doctor.rb

```diff
@@ -92,6 +92,28 @@ def check_for_macgpg2
     end
   end
 
+  # Anaconda installs multiple system & brew dupes, including OpenSSL, Python,
+  # sqlite, libpng, Qt, etc. Regularly breaks compile on Vim, MacVim and others.
+  # Is flagged as part of the *-config script checks below, but people seem
+  # to ignore those as warnings rather than extremely likely breakage.
+  def check_for_anaconda
+    return unless which("anaconda")
+    anaconda = which("anaconda").dirname.realpath
```

Comment:
I just found it should be `realpath.dirname` not the reverse. The order matters.